src/app/models/vm-size.ts

```diff
@@ -50,3 +55,141 @@ export class VmSize extends Record<VmSizeAttributes> {
         super({ ...data, id: data.name && data.name.toLowerCase() } as any);
     }
 }
+
```

Comment:
why is this here? shouldn't that be loaded?

Reply:
This is used for testing or also used when loading failed

Reply:
not a big fan of this. Loading shouldn't fail. I would better show a text box if vm sizes are not loaded

Reply:
We can move this to mock test instead